src/wallet/coin_selection.rs

```diff
@@ -1226,6 +1226,7 @@ mod test {
     }
 
     #[test]
+    #[ignore]
```

Comment:
Is this a test that broke and needs to be fixed later or when new CS module you're working on is ready?